### PR TITLE
Rails 5 compatibility

### PR DESCRIPTION
**Pull request by ebarendt**
Does not include backward compatibility with rails 4.2. I think there is only one method call different, so shouldn't be hard to add back for the short term.

@malept @dcromer

### CODE REVIEW

**Comment by ebarendt** [comment]
Also, most of this PR is Rake 11.2.2 warning fixes, so maybe I can just put that in master.

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,6 @@ gem 'rake'
 group :test do
   gem 'rails'
   gem 'pg'
-  gem 'activerecord', '~> 4.2.0'
```
Comment (malept) [inline Gemfile]: I think this goes in the other commit 😄

**Comment by malept** [comment]
:+1: I'm not too worried about Rails 4.2 compatibility with this gem at the moment. As you said, it's probably pretty easy to add if necessary. However, from a semantic versioning standpoint, this warrants a major version bump. (Adding 4.2 compatibility means it's just a minor version bump.)

**Comment by ebarendt** [comment]
No rails 4.2 is all well and good until we need to patch superstore.

**Comment by malept** [comment]
:shipit:

**Pull request merged**